semgrep/semgrep/cli.py

```diff
@@ -102,51 +128,68 @@ def cli() -> None:
     output.add_argument(
         "-q",
         "--quiet",
-        help="Do not print anything to stdout. Search results can still be saved to an output file specified by -o/--output. Exit code provides success status.",
         action="store_true",
+        help=(
+            "Do not print anything to stdout. Search results can still be "
+            "saved to an output file specified by -o/--output. Exit code "
+            "provides success status."
+        ),
     )
 
     output.add_argument(
         "--no-rewrite-rule-ids",
-        help="Do not rewrite rule ids when they appear in nested subfolders (by default, rule 'foo' in test/rules.yaml will be renamed 'test.foo')",
         action="store_true",
+        help=(
+            "Do not rewrite rule ids when they appear in nested sub-directories "
+            "(by default, rule 'foo' in test/rules.yaml will be renamed "
+            "'test.foo')."
+        ),
     )
 
     output.add_argument(
         "-o",
         "--output",
-        help="Save search results to a file or post to URL. Default is to print to stdout.",
+        help=(
+            "Save search results to a file or post to URL. "
+            "Default is to print to stdout."
+        ),
     )
     output.add_argument(
-        "--json", help="Convert search output to JSON format.", action="store_true"
+        "--json", action="store_true", help="Output results in JSON format."
     )
-    output.add_argument("--test", help="Run a test suite", action="store_true")
+    output.add_argument("--test", action="store_true", help="Run test suite.")
     parser.add_argument(
         "--test-ignore-todo",
-        help="Ignore rules marked as #todoruleid: in test files",
         action="store_true",
+        help="Ignore rules marked as '#todoruleid:' in test files.",
     )
     output.add_argument(
         "--r2c",
-        help="output json in r2c platform format (https://app.r2c.dev)",
         action="store_true",
+        help="Output json in r2c platform format (https://app.r2c.dev).",
```

Comment:
We should link this to the docs (but it's blocked on #637)

Reply:
Roger that, I'll link to the docs after #637 is done :+1: